### PR TITLE
Added square and portrait aspect ratio

### PR DESCRIPTION
**Pull request by guywillis**
Resolves https://github.com/adaptlearning/adapt_framework/issues/3159

* Added support for square and portrait aspect ratio

### CODE REVIEW

File: templates/media.hbs [hunk 1]
```diff
@@ -18,7 +18,7 @@
     {{#if _media.ogg}}
       <audio src="{{_media.ogg}}" type="audio/ogg" style="width: 100%; height: 100%;"/>
     {{else}}
-      <video aria-hidden="true" preload="none" width="640" height="360" {{#if _playsinline}}playsinline{{/if}} {{#equals _media.type "video/vimeo"}}{{else}}poster="{{_media.poster}}"{{/equals}} style="width:100%; height:100%;" controls="controls">
+      <video aria-hidden="true" preload="none" {{#if _isSquare}}width="640" height="640"{{else if _isPortrait}}width="540" height="960"{{else}}width="640" height="360"{{/if}} {{#if _playsinline}}playsinline{{/if}} {{#equals _media.type "video/vimeo"}}{{else}}poster="{{_media.poster}}"{{/equals}} style="width:100%; height:100%;" controls="controls">
```
Comment (eleanor-heath) [inline templates/media.hbs:21]: Can we put some of the if statement logic on separate lines to make this more legible.

**Review by eleanor-heath — APPROVED** [review]

**Review by cahirodoherty-learningpool — APPROVED** [review]
👀

**Review by eleanor-heath — APPROVED** [review]

**Comment by oliverfoster** [comment]
This can be done with one property and an enum. See [`_supportedLayout`](https://github.com/adaptlearning/adapt-contrib-media/blob/e503a8ec9db2d8699cb15289c70f0f01ae180e50/properties.schema#L25-L31)

File: README.md [hunk 2]
```diff
@@ -65,11 +65,8 @@ If using closed captions, use this property to specify which language should be
 ### \_allowFullScreen (boolean):
 Determines whether fullscreen mode is available or not. The default is `false`.
 
-### \_isPortrait (boolean):
-Change the aspect ratio of the video to portrait. The default is `false`.
-
-### \_isSquare (boolean):
-Change the aspect ratio of the video to square. The default is `false`.
+### \_aspectRatio (string):
+Defines the aspect ratio of the video to either be landscape (16:9), portrait (9:16), or square (1:1). Acceptable values are `"full"`, `"left"` or `"right"`. The default is `"landscape"`.
```
Comment (oliverfoster) [inline README.md:69]:
```suggestion
Defines the aspect ratio of the video to either be landscape (16:9), portrait (9:16), or square (1:1). Acceptable values are `"landscape"`, `"portrait"` or `"square"`. The default is `"landscape"`.
```

Comment (guywillis) [reply README.md:69]: Whoops!

**Review by oliverfoster — APPROVED** [review]

**Pull request merged**